corehq/motech/repeaters/models.py

```diff
@@ -197,8 +208,8 @@ def all_ready(self):
 
 
 class SQLRepeater(models.Model):
-    domain = models.CharField(max_length=126)
-    repeater_id = models.CharField(max_length=36)
+    domain = models.CharField(max_length=126, db_index=True)
+    repeater_id = models.CharField(max_length=36, unique=True, db_index=True)
```

Comment:
From the [`unique` docs](https://docs.djangoproject.com/en/2.2/ref/models/fields/#unique):
> Note that when `unique` is `True`, you don’t need to specify `db_index`, because `unique` implies the creation of an index.
```suggestion
    repeater_id = models.CharField(max_length=36, unique=True)
```

Reply:
I did not know that. Thanks for sharing it 😄